src/ontology/mondo-edit.obo

```diff
@@ -584963,6 +584963,21 @@ is_a: MONDO:0003767 {source="NCIT:C50888", source="https://orcid.org/0009-0009-6
 property_value: http://purl.org/dc/terms/creator https://orcid.org/0000-0002-3458-4839
 property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/8003" xsd:anyURI
 
+[Term]
+id: MONDO:1030011
+name: paroxysmal atrial fibrillation
+def: "A disorder characterized by intermittent atrial fibrillation that lasts less than seven days from onset." [https://orcid.org/0009-0009-6791-365X]
+synonym: "paroxymsal AFib" EXACT [NCIT:C80391]
+synonym: "paroxysmal AF" EXACT [NCIT:C80391]
+xref: ICD10CM:I48.0 {source="MONDO:equivalentTo"}
+xref: icd11.foundation:542703670 {source="MONDO:equivalentTo"}
+xref: MESH:D001281 {source="MONDO:relatedTo"}
+xref: NCIT:C80391 {source="MONDO:equivalentTo"}
+xref: SCTID:282825002 {source="MONDO:equivalentTo"}
+is_a: MONDO:0004981 {source="https://orcid.org/0009-0009-6791-365X"} ! atrial fibrillation
```

Comment:
Please use the PMID as a source, and/or some of the x-ref (as appropriate)
Also same question as above about the orcid

Reply:
updated